src/Validator/Validator.php

```diff
@@ -18,7 +18,7 @@ public function __construct(RuleCollection $rules)
         $this->rules = $rules;
     }
 
-    public function validate(DependencyGraph $graph): ViolationCollection
+    public function getErrors(DependencyGraph $graph): ViolationCollection
```

Comment:
The method name `getErrors` is not exactly what you are doing. `Errors != Violations`

Reply:
True, ill change it.